cryptarchia/cryptarchia.py

```diff
@@ -93,20 +143,57 @@ def block_position(self, block: BlockHeader) -> int:
                 return i
 
 
+@dataclass
+class LedgerState:
+    """
+    A snapshot of the ledger state up to some block
+    """
+
+    block: Id = None
+    nonce: Id = None
+    total_stake: int = None
```

Comment:
isn't this a fixed value?

Reply:
it (will be) growing with each block, i.e. when we apply a new block to the ledger state, we will be increasing the total stake by the block reward